common/task/parallelTaskProcessor_test.go

```diff
@@ -210,8 +210,10 @@ func (s *parallelTaskProcessorSuite) TestMonitor() {
 
 	s.processor.shutdownWG.Add(1) // for monitor
 	dcClient := dynamicconfig.NewInMemoryClient()
+	err := dcClient.UpdateValue(dynamicconfig.TaskSchedulerWorkerCount, workerCount)
```

Comment:
yea, this makes sense for fixing tests that had different values before 👍 .  simple and accurate to what prod semantically has to do: overwrite it somehow.